verifyFiles.py

```diff
@@ -279,15 +310,24 @@ def main():
                                              /home/$(USER)/CentOS6.FSL5.0.6
                                              /home/$(USER)/CentOS7.FSL5.0.6
                                              Each directory will contain subject folders like 100307,100308 etc'''))
-        parser.add_argument("-c", "--checksumFile",action="store_true",help="Reads checksum from files. Doesn't compute checksums locally")
+        parser.add_argument("-c", "--checksumFile",help="Reads checksum from files. Doesn't compute checksums locally")
 	parser.add_argument("-d", "--fileDiff", help="Writes the difference matrix into a file")
         parser.add_argument("-m", "--metricsFile", help="CSV file containing metrics definition. Every line contains 4 elements: metric_name,file_extension,command_to_run,output_file_name") 
-        args=parser.parse_args()
+        parser.add_argument("-e","--excludeFolders",nargs='*',help="The list of folders to be ignored while parsing the files and directories")
+	args=parser.parse_args()
         logging.basicConfig(level=logging.INFO,format='%(asctime)s %(message)s')
 	conditions_file_name=args.file_in
         conditions_list=read_conditions_file(conditions_file_name)
         root_dir=os.path.dirname(os.path.abspath(conditions_file_name))
         log_info("Walking through files...")
+	global checksums_flag
+        global exclude_folders
```

Comment:
Please avoid global variables alltogether, they are bad. They make programs more difficult to debug since they could be changed anywhere in the code and they may be masked by local variables.